<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Programm Login</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layout3" >
   <property name="geometry" >
    <rect>
     <x>21</x>
     <y>143</y>
     <width>302</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="buttonHelp" >
      <property name="text" >
       <string>&amp;Help</string>
      </property>
      <property name="shortcut" >
       <string>F1</string>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>96</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOk" >
      <property name="minimumSize" >
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
      <property name="text" >
       <string>&amp;OK</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel" >
      <property name="text" >
       <string>&amp;Cancel</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Q3TextEdit" name="ctmsTextUsername" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>30</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ctmsLblUsername" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>35</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family></family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="ctmsLblPassword" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>85</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Password</string>
   </property>
  </widget>
  <widget class="Q3TextEdit" name="ctmsTextPassword" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>80</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends></extends>
   <header>q3textedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
